CODE-OF-CONDUCT.md

```diff
@@ -13,38 +13,38 @@ Examples of behavior that contributes to a positive environment for our communit
 - Demonstrating compassion and kindness toward other people
 - Being respectful of differing opinions, viewpoints, and experiences
 - Giving and gracefully accepting constructive feedback
-- Accepting responsibility and apologizing to those affected by our mistakes, and learning from the experience
+- Accepting responsibility and apologizing to those affected by our harm and our mistakes, and learning from the experience
 - Focusing on what is best not just for us as individuals, but for the overall community
 
 Examples of unacceptable behavior include:
 
-- Refusal to respect and to use someone's methods of self-identification, (e.g., using someone's pronouns)
-- The use of any terms deemed in the dictionary (but not limited to) as words to avoid as slurs
-- The use of sexualized language or imagery, and sexual attention or advances of any kind
-- Trolling, insulting or derogatory comments, and personal or political attacks
-- Public or private harassment
-- Publishing others' private information, such as a physical or email address, without their explicit permission
-- Other conduct which could reasonably be considered inappropriate in a professional setting
+- refusal to respect and to use someone's methods of self-identification, (e.g., refusing to use someone's pronouns);
+- use of any terms deemed in the dictionary (but not limited to) as words to avoid as slurs;
+- any consentless use of sexualised language or imagery, and sexual attention or advances of any kind;
+- trolling, insulting, or derogatory comments, and personal or political attacks;
+- public or private harassment;
+- publishing others' private information, such as a physical or email address, without their explicit permission;
+- other conduct which could reasonably be considered inappropriate in a professional setting
 
 ## Enforcement Responsibilities
 
-Community leaders are responsible for clarifying and enforcing our standards of acceptable behavior and will take appropriate and fair corrective action in response to any behavior that they deem inappropriate, threatening, offensive, or harmful.
+Community leaders and moderators are responsible for clarifying and enforcing our standards of acceptable behavior and will take appropriate and fair corrective action in response to any behavior that they deem inappropriate, threatening, offensive, or harmful.
```

Comment:
I think we can change the discord `maintainers` role to be `community-leaders`, and then stick with the "community leaders" language by itself. Thoughts?